spec/index.bs

```diff
@@ -703,12 +703,20 @@ This specification defines the following [=usage scopes=]:
   2. an individual command on a {{GPUComputePassEncoder}}, such as {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}.
   3. the whole {{GPURenderPassEncoder}}.
 
-Note: calling {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}
-adds the {{GPUBindGroup/[[usedResources]]}} to the [=usage scope=]
-regardless of whether the shader or {{GPUPipelineLayout}} actually depends on these bindings.
-Similarly {{GPURenderEncoderBase/setIndexBuffer()|GPURenderEncoderBase.setIndexBuffer}}
-add the index buffer to the usage scope (as [=internal usage/input=])
-regardless of whether the indexed draw calls are used afterwards.
+<div class=note>
+  Resource usages are tracked at state-setting time, and they ignore binding {{GPUBindGroupLayoutEntry/visibility}}.
+  This means the following example usages within [=usage scopes=] **are** tracked and validated:
+
+  - Bind groups, vertex buffers or index buffers not used by any pipelines or any draw calls. For example,
+  calling {{GPUProgrammablePassEncoder/setBindGroup(index, bindGroup, dynamicOffsets)|GPUProgrammablePassEncoder.setBindGroup}}
+  adds the {{GPUBindGroup/[[usedResources]]}} to the [=usage scope=]
+  regardless of whether the shader or {{GPUPipelineLayout}} actually depends on these bindings.
+  Similarly {{GPURenderEncoderBase/setIndexBuffer()|GPURenderEncoderBase.setIndexBuffer}}.
+  add the index buffer to the usage scope (as [=internal usage/input=])
+  regardless of whether the indexed draw calls are used afterwards.
+  - Bind groups, index buffers, or vertex buffers which are `set`, then shadowed by another `set` call.
```

Comment:
isn't this a subset of the previous item on this list?

Reply:
Done. Removed this one.